src/fsw/FCCode/ADCSBoxMonitor.cpp

```diff
@@ -8,23 +8,23 @@ ADCSBoxMonitor::ADCSBoxMonitor(StateFieldRegistry &registry,
     unsigned int offset, Devices::ADCS &_adcs)
     : TimedControlTask<void>(registry, "adcs_monitor", offset),
     adcs_system(_adcs),
-    rwa_speed_rd_sr(adcs::rwa::min_speed_read, adcs::rwa::max_speed_read, 16*3), //referenced from I2C_Interface.doc
+    rwa_speed_rd_sr(adcs::rwa::min_speed_read, adcs::rwa::max_speed_read, 8*3), //full res is 16*3
     rwa_speed_rd_f("adcs_monitor.rwa_speed_rd", rwa_speed_rd_sr),
-    rwa_torque_rd_sr(adcs::rwa::min_torque, adcs::rwa::max_torque, 16*3), //referenced from I2C_Interface.doc
+    rwa_torque_rd_sr(adcs::rwa::min_torque, adcs::rwa::max_torque, 8*3), //full res is 16*3
     rwa_torque_rd_f("adcs_monitor.rwa_torque_rd", rwa_torque_rd_sr),
-    ssa_mode_rd(0,2,2), //referenced from Interface.doc
+    ssa_mode_rd(0,2,2), //3 modes
     ssa_mode_f("adcs_monitor.ssa_mode", ssa_mode_rd),
-    ssa_vec_sr(-1,1,16*3), //referenced from I2C_Interface.doc
+    ssa_vec_sr(0,1,1 + SerializerConstants::min_fvsz), //full res is 16*3
     ssa_vec_f("adcs_monitor.ssa_vec", ssa_vec_sr),
-    ssa_voltage_sr(adcs::ssa::min_voltage_rd, adcs::ssa::max_voltage_rd, 8),
+    ssa_voltage_sr(adcs::ssa::min_voltage_rd, adcs::ssa::max_voltage_rd, 8), //per voltage, full is 8
```

Comment:
What is the rationale for 8 bits?

Reply:
Discussed offline. Going to change this back to 12 bits because (a) it's not really stuff that we downlink unless something is wrong, so for the most part changing this value doesn't affect telemetry, (b) to enable doing "science" with the values if we so choose.

```suggestion
    ssa_voltage_sr(adcs::ssa::min_voltage_rd, adcs::ssa::max_voltage_rd, 12), // Full resolution to enable scientific analysis of the sun sensing algorithm
```